.github/workflows/ci-links.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: '18.x'
+          node-version: '20.x'
```

Comment:
could go to the newest LTS 22.x. in other places we use the oldest supported LTS to test for compatibility, but this is just us building our docs, so newest is fine I think

Reply:
Yeah, I don't see why we shouldn't update across the board as Node 18 is getting dropped soon.

Reply:
oh does netlify run the build for us?

Reply:
Yeah.

Reply:
> oh does netlify run the build for us?

The preview pages are rendered on netlify for us. They use a separate stack running Ubuntu 20.04 and Node version up to 20.x.